flink-python/pyflink/common/types.py

```diff
@@ -287,26 +293,51 @@ def __len__(self):
 
 def to_java_data_structure(value):
```

Comment:
Will this method only used in test cases? If so, we could move it files such as test_case_utils.py or some other place.